databases/week3/WEEK3 homework/mealSharing_db

```diff
@@ -0,0 +1,256 @@
+--following queries
+
+
+
+
+
+
+INSERT INTO Meal(title,description,location,Hosting_time,max_reservations,price,created_date)
+Values('Ethiopian Injera and Tibs ','Injera, a traditional, fermented, spongy Ethiopian bread made with Teff flour','Addis Ababa, Ethiopia ','2022-09-10 17:50:20',10,50,'2022-07-20'),
+	(' Vegan Pesto Pasta ','Cavatelli Pasta-Fresh Pesto','Noto, Italy','2022-03-10 17:30:20',15,40,'2022-02-20'),
+    (' Samosa with Green Chutney','Samosa-Green Chutney (coriander dipping sauce)','Jaipur, India ','2022-04-10 17:30:20',20,30,'2022-03-20');
+
+
+
+        INSERT INTO Reservation(number_of_guests,meal_id,created_date,contact_phonenumber,contact_name,contact_email) 
+VALUES(2,7,'2022-07-28','44 67850238','Angi','Angi@databse.dk'),
+      (6,7,'2022-07-28','44 67850238','Saron','Sarii@databse.dk'),
+      (4,8,'2022-02-28','44 67850238','Maria','Mar@databse.dk'),
+      (7,9,'2022-04-28','44 67850238','Mongo','Monk@databse.dk'),
+      (8,9,'2022-04-28','44 67850238','Mik','Mikke@databse.dk');
+
+
+      INSERT INTO Review(title,description,meal_id,stars,created_date)
+VALUES('Ethiopian Injera'," We’ll definitely do this again",7,4,'2021-02-26'),
+       ('Ethiopian food',"We had a wonderful time, learned about Ethiopian food and culture and had an amazing meal",7,5,'2021-02-26'), 
+       ('Injera food',"Thank you so much for this nice experience",7,5,'2021-02-26'), 
+       ('Vegan Pesto Pasta ',"It was a fun and unique experience! Thank you",8,4,'2021-02-26'), 
+       (' Pesto Pasta '," I believe my white flour would have been better.",8,3,'2021-02-26'), 
+       ('Vegan Pesto',"We had a great time. They were fantastic hosts ",8,4,'2021-02-26'), 
+       ('Samosa with Green Chutney',"I got it as a gift for my daughter and her boyfriend because they love Smosa",9,5,'2021-02-26'), 
+       ('Samosa ',"great meal",9,5,'2021-02-26'), 
+       ('Samosa with Chutney'," an amazing experience and we’ll be back for sure.",9,5,'2021-02-26');
+
+
+    --Functionality
+-- Get meals that has a price smaller than a specific price fx 90
+
+SELECT*
+ FROM Meal
+ WHERE price <50;
+-- Get meals that still has available reservations
+Select SUM(number_of_guests) AS reserved,meal_id,Meal.title,max_reservations
+FROM Reservation 
+JOIN Meal On Meal.id= Reservation.meal_id
+-- Where Reservation.reserved < Meal.max_reservations
+-- I tried to filter the result using the line above , but it did not worked
+GROUP BY meal_id;
+--Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+
+SELECT * 
+FROM Meal
+WHERE title LIKE '%south indian%';
+--Get meals that has been created between two dates
+SELECT*
+FROM Meal
+WHERE created_date >'2022-03-20' AND created_date < '2022-06-20';
+-- Get only specific number of meals fx return only 5 meals
+SELECT * 
+FROM Meal
+LIMIT 3;
+--Get the meals that have good reviews
+
+Select avg(stars),meal_id,Meal.title
```

Comment:
Why are you using average stars here? o.O

Reply:
so I got average of stars Group by meal id , to get the meals average review point.

Reply:
Yes, but the exercise only asks for: --Get the meals that have good reviews. Take it easy ;p